contracts/registries/LicenseRegistry.sol

```diff
@@ -112,14 +129,16 @@ contract LicenseRegistry is ILicenseRegistry, ERC1155, Governable {
             transferable: transferable
         });
         bool isNew;
+        LicenseRegistryStorage storage $ = _getLicenseRegistryStorage();
+
         (licenseId, isNew) = DataUniqueness.addIdOrGetExisting(
             abi.encode(licenseData),
-            _hashedLicenses,
-            _mintedLicenses
+            $.hashedLicenses,
+            $.mintedLicenses
         );
         if (isNew) {
-            _mintedLicenses = licenseId;
-            _licenses[licenseId] = licenseData;
+            $.mintedLicenses = licenseId;
```

Comment:
same question, new coding style of using "$", instead of _ prefix?

Reply:
_prefix is for internal or private variables, $ is for the storage struct for the contract